client-react/src/signin/index.js

```diff
@@ -152,7 +171,10 @@ const SignIn = () => {
       </ContentWrapper>
       <ContentWrapper display={page[1] ? true : false}>
         <LeftContainer smol>
-          <Title light>Welcome!</Title>
+          <Branding style={{ marginTop: "-90px" }} src={logo} alt="logo" />
+          <Title light style={{ marginTop: "60px" }}>
```

Comment:
rather than doing inline styling. lets pass in a prop and let styled-components handle which style needs to be applied